packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -488,11 +490,32 @@ export class JobService {
     skip: number,
     limit: number,
   ): Promise<EscrowData[]> {
-    const escrows = await EscrowUtils.getEscrows({
-      networks,
-      jobRequesterId: userId.toString(),
-      status: filterToEscrowStatus(status),
-    });
+    const escrows: EscrowData[] = [];
+    escrows.push(
+      ...(await EscrowUtils.getEscrows({
+        networks,
+        jobRequesterId: userId.toString(),
+        status: filterToEscrowStatus(status),
+      })),
```

Comment:
You need to pass the launcher address otherwise you get all other launchers jobs
```suggestion
      ...(await EscrowUtils.getEscrows({
        networks,
        launcher: launcherAddress,
        jobRequesterId: userId.toString(),
        status: filterToEscrowStatus(status),
      })),
```